build_tools/spec/region_spec.rb

```diff
@@ -41,7 +42,9 @@ def whitelist
 
           # skip known whitelists
           next if whitelist[key] &&
-                  (whitelist[key][File.basename(path)] == idx ||
+                  (((idx - lines_inserted)..idx).include?(
+                    whitelist[key][File.basename(path)]
+                  ) ||
```

Comment:
Would this change still be necessary? When the files are committed with the magic comment, all of the numbers above will need a +2 (see that you're doing it dynamically here)

Reply:
No, it does not.  
This should be removed before merging. It is required for now to make the hack work.